dist. loads / hydrostatics callout cards

agyao2 opened the pull request:
Addresses this [this issue.](https://github.com/IllinoisRef/mechref/issues/248)

mrhoyle2 reviewed (changes requested):
"distributed loads reference page" should be all lowercase. Please add the heads up card on the dist loads page as well.

agyao2 commented:
> The text "reference page" should be lower case as well.
> 
> I had to reload this branch like 3 times to get the dist. loads card to render. For that card, can you change the linked text to "Hydrostatic fluid pressure" and change the ending to "later in statics" or something similar so its clearer that its in the same course?



> The text "reference page" should be lower case as well.
> 
> I had to reload this branch like 3 times to get the dist. loads card to render. For that card, can you change the linked text to "Hydrostatic fluid pressure" and change the ending to "later in statics" or something similar so its clearer that its in the same course?

Fixed everything, turns out the link was going to the actual mechref page instead of the local page which is why the dist. load heads up card wouldn't show up so I fixed that
